### ISSUE TITLE
Dimelo SDK image selection issue

### ISSUE THREAD

**Issue by shadabhub** [dimelo/Dimelo-Android#37]
When we select any image for attachment we are getting below error

Error:
java.lang.OutOfMemoryError: Failed to allocate a 48771084 byte allocation with 16777216 free bytes and 41MB until OOM
                                                                          at dalvik.system.VMRuntime.newNonMovableArray(Native Method)
                                                                          at android.graphics.BitmapFactory.nativeDecodeByteArray(Native Method)
                                                                          at android.graphics.BitmapFactory.decodeByteArray(BitmapFactory.java:613)
                                                                          at android.graphics.BitmapFactory.decodeByteArray(BitmapFactory.java:636)
                                                                          at com.dimelo.dimelosdk.helpers.Image.ImageCompressor.compress(ImageCompressor.java:22)
                                                                          at com.dimelo.dimelosdk.main.StartAndValidateAttachmentProvider$1.onClick(StartAndValidateAttachmentProvider.java:217)
                                                                          at android.view.View.performClick(View.java:6308)
                                                                          at android.view.View$PerformClick.run(View.java:23969)
                                                                          at android.os.Handler.handleCallback(Handler.java:751)
                                                                          at android.os.Handler.dispatchMessage(Handler.java:95)
                                                                          at android.os.Looper.loop(Looper.java:154)
                                                                          at android.app.ActivityThread.main(ActivityThread.java:6823)
                                                                          at java.lang.reflect.Method.invoke(Native Method)
                                                                          at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1563)
                                                                          at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1451)

**Comment by waelba** [comment]
Hi @shadabhub, 

Your image is so big, so add these lines to your `androidManifest` file :

```
<application
    android:hardwareAccelerated="false"
    android:largeHeap="true"
    ....
>
```

**Comment by ElizavetaGorbunova** [comment]
Hi @waelbenabdallah! I've got the same issue. Unfortunately, I can't increase a heap of the app and disable `hardwareAccelerated `flag, because it'll break other non-Dimelo functionality. Is there another solution to this? For example, compress image if it is too large. Could you take a look for this issue one more time, please?

**Comment by waelba** [comment]
Hi @shadabhub @MrsBlue,

It will be fixed in the next version `v1.6.9`.

**Comment by waelba** [comment]
Hello @shadabhub,

You can test it on `v1.6.9` now.
